src/modules/70-pipeline/components/PipelineSteps/Steps/CustomVariables/CustomVariablesEditableStage.tsx

```diff
@@ -133,7 +133,7 @@ export function CustomVariablesEditableStage(props: CustomVariableEditableProps)
                       <Text font={{ variation: FontVariation.TABLE_HEADERS }}>{getString('valueLabel')}</Text>
                     </div>
                   ) : null}
-                  {values.variables.map?.((variable, index) => {
+                  {values.variables.map?.((variable: any, index: number) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232293)
```suggestion
                  {values.variables.map?.((variable: unknown, index: number) => {
```